Websites/planet.webkit.org/config.ini

```diff
@@ -45,14 +45,14 @@ new_date_format = %B %d, %Y
 encoding = utf-8
 # locale = C
 
-
 # To define a different value for a particular template you may create
 # a section with the same name as the template file's filename (as given
 # in template_files).
 
 # Provide no more than 7 days articles on the front page
 [templates/index.html.tmpl]
-days_per_page = 14
+# Setting this to 10 years
```

Comment:
This comment says what the following line does, but not why. It's not explained in the commit message either.